### ISSUE TITLE
Remove title and description from right arrow image on "read more" link

### ISSUE THREAD

**Issue by maya** [18F/18f.gsa.gov#2221]
The right arrow image on the "read more" link is decorative so I don't believe you need title and description. Announcing "Read more about [title of post]" will be the most helpful text for someone using a screenreader.

"about [title of post]" can be added as screenreader only text, which I believe is what it was like before, but @gboone may remember better.

/cc @nickbristow

**Comment by gemfarmer** [comment]
@maya Because these arrows are used site wide to indicated 'read more', do you think a compromise could be to make the title `Read more` instead of `right arrow image icon`? Or do you think it is still confusing having that help text in conjunction with the other content in that card anchor tag?

**Comment by maya** [comment]
@gemfarmer nope, bc that would be duplicate content bc the link already says "Read more." https://www.w3.org/WAI/tutorials/images/decorative/

**Comment by nickbristow** [comment]
@gemfarmer yeah, I agree with @maya "arrow right" doesn't add anything and if there is text, its duplicate content and not needed.

**Comment by gemfarmer** [comment]
@maya @nickbristow that tutorial is helpful. I went back through using VoiceOver and I see what both of you mean about the duplication. In that audit, I found the Featured Projects card links even more troubling. Do either of you know an effective strategy for putting a link around an element that has a lot of content without reading all of the internal content?

**Comment by maya** [comment]
@gemfarmer that's a good question! I asked in the [a11y slack](https://web-a11y.herokuapp.com/) and got this answer:

>Personally I just add a click event handler on the whole card but don't make it a tab stop and then just focus on the real link for keyboard/screen reader users. just treat the click event handler as a redundant way of doing the same thing as the link

**Comment by nickbristow** [comment]
@gemfarmer yeah, having a ton of content in a link isn't really a great for accessibility. Click handler on the card, and just having the link be focusable is probably the way i'd go
